src/Augmint.ts

```diff
@@ -30,10 +30,7 @@ export class Augmint {
         Object.keys(stubs).forEach(stub => {
             const role = stub;
             const contractListStub = stubs[stub];
-            deployedEnvironment.addRole(
-                role,
-                contractListStub.map(contractStub => new DeployedContract(contractStub))
-            );
+            deployedEnvironment.addRole(role, contractListStub.map(contractStub => new DeployedContract(contractStub)));
```

Comment:
is this some autoformatting? did the settings change?

Reply:
yep i've noticed. it's happened before too.  I suspect that @rszaloki  doesn't have prettier format on save set or something else different in our editor. We should make sure we sync our setting